integration_tests/test_str_attributes.py

```diff
@@ -25,6 +25,21 @@ def strip():
     assert "     AASAsaSas    " .lstrip() == "AASAsaSas    "
     assert "     AASAsaSas    " .strip() == "AASAsaSas"
 
+def startswith():
+    s: str
+    s = "   empty"
+    assert s.startswith("  ") == True
+    assert "  @" .startswith(" ") == True
+    assert "   emptyAaaaAABBbbbbBB" .startswith(s) == True
```

Comment:
Let's try giving something in the middle portion of the string such as `AABB` in `startswith` and see if it returns `False`. Also give something like `emptyAaaX` as input to `startswith`.

Reply:
i added them and edit those corner cases in functions, i think it's ready now.